src/components/card/MyReservationCard.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from "next/image";
+import { HTMLAttributes } from "react";
+
+import { cn } from "@/utils/classNames";
+import { formatNumber } from "@/utils/common";
+
+interface MyReservationCardProps extends HTMLAttributes<HTMLDivElement> {
+  activity: {
+    bannerImageUrl: string;
+    title: string;
+  };
+  status: "pending" | "confirmed" | "declined" | "canceled" | "completed";
+  reviewSubmitted: boolean;
+  totalPrice: number;
+  headCount: number;
+  date: string;
+  startTime: string;
+  endTime: string;
+}
+
+/**
+ * 예약관리 리스트 아이템
+ */
+export default function MyReservationCard({
+  status,
+  activity,
+  date,
+  startTime,
+  endTime,
+  totalPrice,
+  headCount,
+  reviewSubmitted,
+  className = "",
+}: MyReservationCardProps) {
+  /* TODO: Button component 반영 */
+  function ReviewButtons() {
+    return reviewSubmitted ? (
+      <>
+        <button className="w-[100%]">예약 변경</button>
```

Comment:
```suggestion
        <button className="w-full">예약 변경</button>
```
`w-[100%]`는 `w-full`로 이미 지정된 클래스를 활용하는 것이 좋을 것 같아요 🤔